### PR TITLE
Change image tags for release 1.4-rc1

### PR DESCRIPTION
**Pull request by nachtmaar**
**Description**

Changes proposed in this pull request:

- Change image tags for release 1.4-rc1

**Related issue(s)**

ToDo:
- [ ] enable `rebase and merge option` on kyma repo

### CODE REVIEW

File: resources/cms/values.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 global:
   containerRegistry:
     path: eu.gcr.io/kyma-project
-  cmsControllerManager:
-    dir: develop/
-    version: '5bad2085'
+  cms_controller_manager:
```
Comment (magicmatatjahu) [inline resources/cms/values.yaml]: Please don't change name! https://github.com/kyma-project/kyma/blob/release-1.4/resources/cms/charts/cms-controller-manager/templates/statefulset.yaml#L28

Comment (nachtmaar) [reply resources/cms/values.yaml]: Good catch! Thank you

**Pull request merged**